src/components/YouTube.tsx

```diff
@@ -27,7 +22,7 @@ const YouTube: React.FC<IProps> = ({ id, start = "0", title = "YouTube" }) => {
   const baseUrl = "https://www.youtube.com/embed/"
   const src = baseUrl + id + startQuery
   return (
-    <Figure>
+    <Box as="figure" display="block" marginY="1rem">
```

Comment:
```suggestion
    <Box as="figure" my={4}>
```
- `figure` is by default `block` displayed, we can ignore it
- there is a short version of `marginY` => `my`
- lets try to use the chakra spacing scale. 1 rem = 4 (more about this [here](https://chakra-ui.com/docs/styled-system/theme#spacing))

Reply:
Yes, we are going to use the default values. In the Design System too.